packages/server/src/auth/utils/cookies.ts

```diff
@@ -21,13 +22,24 @@ export function getAuthCookies(request: GenericRequest): AuthMaterialBasic {
     return { jwt, refreshToken };
 }
 
-export function setAuthCookies(response: GenericResponse, authMaterial: AuthMaterial) {
+export function setAuthCookies(response: GenericResponse, authMaterial: AuthMaterial, options?: CookieOptions) {
     const cookies = [
-        createCookieString({ name: SESSION_PREFIX, value: authMaterial.jwt }),
+        createCookieString({
+            name: SESSION_PREFIX,
+            value: authMaterial.jwt,
+            options: {
+                ...options,
+                // The JWT needs to be accessible to the browser for the client SDK to work
+                httpOnly: false,
```

Comment:
I understand this, but should we document it in the README?
Otherwise it might be confusing for the developer to configure `httpOnly: true` on the `CrossmintAuth` options and then see it `false` in their debugging sessions.